src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_acl.erl

```diff
@@ -254,7 +254,12 @@ names_to_ids(ACE, OrgId) ->
             safe_fetch_ids(user, undefined, ej:get({<<"users">>}, ACE)) ++
             safe_fetch_ids(client, OrgId, Clients)
     end,
-    GroupIds = safe_fetch_ids(group, OrgId, ej:get({<<"groups">>}, ACE)),
+    %% Global groups can be intermixed with org local groups. This creates some ambiguity as
```

Comment:
I do see a potential problem here.  The scenario is where someone had permissions assigned to a same-named group within the org before the migration ran that created the global group.  The first time that they PUT back a body containing the global group , the org group would lose permissions.  It's obscure and not likely to be hit by many people - but for those who do hit it, it's   unavoidable if we don't enforce a policy preventing the assignment similar to our user/client disambiguation.